project/Tasks.scala

```diff
@@ -5,15 +5,17 @@ import sbt._
 
 object Generators {
   // Generates a scala file that contains the Lagom version for use at runtime.
-  def version(lagomVersion: String, dir: File): Seq[File] = {
+  def version(lagomVersion: String, akkaVersion: String, playVersion: String, dir: File): Seq[File] = {
     val file = dir / "com" / "lightbend" / "lagom" / "core" / "LagomVersion.scala"
     val scalaSource =
-      """|package com.lightbend.lagom.core
-         |
-         |object LagomVersion {
-         |    val current = "%s"
-         |}
-          """.stripMargin.format(lagomVersion)
+      s"""|package com.lightbend.lagom.core
+          |
+          |object LagomVersion {
+          |  val current = "$lagomVersion"
+          |  val akka = "$akkaVersion"
```

Comment:
I think we should add Akka HTTP too (but no more).  Some downstream projects (aka `play-grpc`) require some modules of Akka HTTP which Lagom doesn't depend on transitively.